webapp/packages/core-blocks/src/ToolsPanel/ToolsPanel.m.css

```diff
@@ -1,5 +1,7 @@
 .toolsPanel {
+  composes: theme-background-surface theme-text-on-surface theme-border-color-background from global;
+  border-bottom: solid 1px;
   height: 30px;
   display: flex;
-  flex-shrink: 0;
+  flex: 0 0 auto;
```

Comment:
leads to unwanted changes in some uses of ToolsPanels
<img width="1534" alt="image" src="https://github.com/dbeaver/cloudbeaver/assets/811729/55c39678-db85-42b5-b12d-47b8266c1a5c">